### ISSUE TITLE
Some katas with removed comments are broken

### ISSUE THREAD

**Issue by FArekkusu** [Codewars/codewars.com#1806]
Some of the katas which received racist comments from cheaters/spammers ([1](https://github.com/Codewars/codewars.com/issues/1493#issuecomment-475952839) and [2](https://github.com/Codewars/codewars.com/issues/1493#issuecomment-476951686)) have corrupt data now (as it has happened recently with the "who's going to pay for the wall" kata), and cannot be solved. It is unknown which exact katas are not working anymore, but at least one of them has been receiving comments about being broken for weeks already, ["Decode the Morse code, for real"](https://www.codewars.com/kata/54acd76f7207c6a2880012bb/discuss).

**Comment by kazk** [comment]
This should be a duplicate of #1788.

I knew deleting data on Codewars can be problematic, but I underestimated it. Next time, I'll change the comment to `[deleted]` or something instead of actually deleting.

The data of "Who's going to pay for the wall" is really broken now because I had to remove some stuff for #1801. The data for its solution pages are gone, but individual solutions are there so it should be possible to rebuild. I just haven't had the time to figure it out.
